### PR TITLE
Add start time, end time, and date to each Place card

### PR DESCRIPTION
**Pull request by sebastianbarry**
infinite_scroll_list.js
- Increased the default height of the place cards in getCardHeight() by 82px

place_list_item.html
- Added place start time tag
- Added place display date to be centered above the place name
- Added place end time tag

### CODE REVIEW

**Comment by sebastianbarry** [comment]
Testing:

Simulator Screen Shot - iPhone 11 - 2023-03-28 at 22 38 36

Simulator Screen Shot - iPhone 11 - 2023-03-28 at 23 06 24

**Comment by sebastianbarry** [comment]
@JGreenlee @shankari could  you take a quick look and make sure this seems correct with what UNSW asked for?

> start time, end time, and date to each Place card, similar to what we have on Trip cards

**Comment by sebastianbarry** [comment]
ToDo: 
Make sure this solution is inline with @MaliheTabasi comment

1. Make the end time of the last place be the end of the day
2. If there are multiple days in between 2 trip objects, there will be place cards to fill in for each day: https://github.com/e-mission/e-mission-docs/issues/871#issuecomment-1486148982

File: www/js/diary/infinite_scroll_list.js [hunk 1]
```diff
@@ -320,6 +320,12 @@ angular.module('emission.main.diary.infscrolllist',['ui-leaflet',
         $scope.data.displayTimelineEntries.push(place);
       }
     });
+
+    // if we are showing places and the last place does not have a display_end_time, 
+    // then we will use the display_start_time + 1 hour for display_end_time
+    if (!$scope.data.displayTimelineEntries[$scope.data.displayTimelineEntries.length-1].display_end_time && $scope.showPlaces) {
+      $scope.data.displayTimelineEntries[$scope.data.displayTimelineEntries.length-1].display_end_time = moment($scope.data.displayTimelineEntries[$scope.data.displayTimelineEntries.length-1].enter_fmt_time).parseZone().add(1, 'hour').format("h:mm A");
```
Comment (shankari) [inline www/js/diary/infinite_scroll_list.js]: If you see the discussion on https://github.com/e-mission/e-mission-docs/issues/871, I think that they expect it to be the end of the day.

**Review by shankari — CHANGES_REQUESTED** [review]
Please read the actual request from https://github.com/e-mission/e-mission-docs/issues/871 for context.

**Comment by shankari** [comment]
@sebastianbarry I would suggest that you hold off on the change below

> If there are multiple days in between 2 trip objects, there will be place cards to fill in for each day: https://github.com/e-mission/e-mission-docs/issues/871#issuecomment-1486148982

pending the discussion in 
https://github.com/e-mission/e-mission-docs/issues/871#issuecomment-1488774436

Please do fix my comment above and indicate when this is ready to merge.

File: www/js/diary/infinite_scroll_list.js (hunk 1 above)
Comment (sebastianbarry) [inline www/js/diary/infinite_scroll_list.js]:
```suggestion
      $scope.data.displayTimelineEntries[$scope.data.displayTimelineEntries.length-1].display_end_time = moment($scope.data.displayTimelineEntries[$scope.data.displayTimelineEntries.length-1].enter_fmt_time).parseZone().endOf('day').format("h:mm A");
```

**Comment by sebastianbarry** [comment]
Addressed comment https://github.com/sebastianbarry/e-mission-phone/pull/8#discussion_r1151407719 in commit https://github.com/sebastianbarry/e-mission-phone/pull/8/commits/e521390b94ff8d8420f5faededfdcf2dca5b3e90

Testing: 
Simulator Screen Shot - iPhone 11 - 2023-03-29 at 09 16 57

Holding off on the other change, as indicated in your message @shankari. This should be ready to merge then

**Comment by shankari** [comment]
@sebastianbarry we are deploying code from https://github.com/e-mission/e-mission-phone/tree/survey_refactor_expansion
to staging. I am not sure where this code is going...

In general, I review code and then merge it...

**Pull request merged**